docs/_articles/en/einstein/einstein-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Einstein for Developers Overview
+lang: en
+---
+
+Einstein for Developers is an AI-powered developer tool that is available as an easy-to-install VS Code extension built using CodeGen, Salesforce’s secure, custom AI model. Use the tool to enhance developer productivity by quickly generating code suggestions using natural language instructions. The extension is available in the [VS Code](https://marketplace.visualstudio.com/vscode) and [Open VSX](https://open-vsx.org/) marketplaces.
```

Comment:
Should we restructure these two sentences a bit to focus on what value we give to the user first and then what technologies the product is built on?